pages/account/manage/profile.js

```diff
@@ -232,6 +236,14 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
                       </p>
                     </div>
                   </div>
+                  <div className="mt-3">
+                    <Toggle
+                      text1="Enable?"
+                      text2="hide/show profile view and rank"
```

Comment:
This is the first time `text2` is being used, so the test is detecting not a high enough colour contrast. This can be fixed here 

https://github.com/EddieHubCommunity/BioDrop/blob/5d0376fb0576016cda440aa0d90905b2ddd7c4f6/components/form/Toggle.js#L37

```diff
- text-primary-medium-low
+ text-primary-high dark:text-primary-low
```